dolphinscheduler-spi/src/main/java/org/apache/dolphinscheduler/spi/common/Constants.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.spi.common;
+
+public class Constants {
+
+    /**
+     * task log info format
+     */
+    public static final String TASK_LOG_INFO_FORMAT = "TaskLogInfo-%s";
```

Comment:
This variable exists in org.apache.dolphinscheduler.spi.task.TaskConstants

Reply:
done, thx for your remind